aws-kendra-datasource/src/main/java/software/amazon/kendra/datasource/convert/cde/DocumentAttributeConditionConverter.java

```diff
@@ -0,0 +1,30 @@
+package software.amazon.kendra.datasource.convert.cde;
+
+import software.amazon.awssdk.services.kendra.model.DocumentAttributeCondition;
+
+public class DocumentAttributeConditionConverter {
+
+  public static software.amazon.awssdk.services.kendra.model.DocumentAttributeCondition toSdkDocumentAttributeCondition(
+      final software.amazon.kendra.datasource.DocumentAttributeCondition documentAttributeCondition) {
+    if (documentAttributeCondition == null) {
+      return null;
+    }
+    return DocumentAttributeCondition.builder()
+        .conditionDocumentAttributeKey(documentAttributeCondition.getConditionDocumentAttributeKey())
+        .operator(software.amazon.awssdk.services.kendra.model.ConditionOperator.fromValue(documentAttributeCondition.getOperator()))
```

Comment:
If .getOperator() is null, what will happen here?

Reply:
.fromValue() handles that, if argument is null, .fromValue() returns null;

Reply:
also operator is a required field for a DocumentAttributeCondition in the json schema so it shouldn't happen.